flink-filesystem-join-redis/initialize_redis.py

```diff
@@ -61,7 +65,22 @@
         keys=["item_id"],
     )
 
-    result_table = client.get_features(item_attributes_source)
+    item_attributes_view = DerivedFeatureView(
```

Comment:
Would it be better to use `item_attributes_view` for consistency with the table name?